From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Groldi <chniucg@gmail.com>
Date: Fri, 11 Aug 2023 23:40:04 +0200
Subject: [PATCH] Refactor localization methods in KissenPaper server
 implementation

The KissenPaperLocalizationImplementation.java file was refactored for better understanding and easier maintenance. The comments were adjusted, the unknown command situations have been reordered within the spigot section, and a necessary explanation for getTranslation method has been added. This helps other developers know what this method does more conveniently.

diff --git a/src/main/java/net/kissenpvp/paper/message/localization/KissenPaperLocalizationImplementation.java b/src/main/java/net/kissenpvp/paper/message/localization/KissenPaperLocalizationImplementation.java
index cf3730e36c870b32fdbd757928029ede72fac7aa..49cede20d40758bd9650bc0f41340d84f22978f4 100644
--- a/src/main/java/net/kissenpvp/paper/message/localization/KissenPaperLocalizationImplementation.java
+++ b/src/main/java/net/kissenpvp/paper/message/localization/KissenPaperLocalizationImplementation.java
@@ -11,24 +11,30 @@ public class KissenPaperLocalizationImplementation extends KissenLocalizationImp
     @Override
     public boolean postStart() {
 
-        //Bukkit
+        // bukkit
         register("bukkit.shutdown-message", getTranslation("multiplayer.disconnect.server_shutdown"));
 
-        //Spigot
+        // spigot
         register("spigot.whitelist", getTranslation("multiplayer.disconnect.not_whitelisted"));
-        register("spigot.unknown-command", new MessageFormat("Unknown command. Type \"/help\" for help."));
         register("spigot.server-full", getTranslation("multiplayer.disconnect.server_full"));
         register("spigot.outdated-client", getTranslation("multiplayer.disconnect.outdated_client"));
         register("spigot.outdated-server", getTranslation("multiplayer.disconnect.outdated_server"));
         register("spigot.restart", getTranslation("multiplayer.disconnect.server_shutdown"));
+        register("spigot.unknown-command", new MessageFormat("Unknown command. Type \"/help\" for help."));
 
-        //Paper
+        // paper
         register("paper.kick.connection-throttle", new MessageFormat("Connection throttled! Please wait before reconnecting."));
         register("paper.unauthorized", new MessageFormat("You do not have the permission to perform this command."));
 
         return super.postStart();
     }
 
+    /**
+     * Retrieves a translated message format based on the provided language key.
+     *
+     * @param languageKey the string representing the language key for the translation
+     * @return the translated message format
+     */
     private MessageFormat getTranslation(String languageKey) {
         return new MessageFormat(KissenComponentSerializer.getInstance()
                 .getMiniSerializer()
